classes/migration/upgrade/v3_4_0/I7901_Duplicate_OAI_IDs.php

```diff
@@ -24,26 +24,24 @@ public function up(): void
         switch (DB::getDriverName()) {
             case 'mysql':
                 DB::unprepared(
-                    "DELETE dot
+                    'DELETE dot
                     FROM data_object_tombstones dot
                     JOIN submissions s ON (dot.data_object_id = s.submission_id)
                     JOIN journals j ON (j.journal_id = s.context_id)
                     JOIN publications p ON (s.current_publication_id = p.publication_id)
-                    JOIN publication_settings psissue ON (psissue.publication_id = p.publication_id AND psissue.setting_name='issueId' AND psissue.locale='')
-                    JOIN issues i ON (CAST(i.issue_id AS CHAR(20)) = psissue.setting_value)
-                    WHERE i.published = 1 AND j.enabled = 1 AND p.status = 3"
+                    JOIN issues i ON (i.issue_id = p.issue_id)
```

Comment:
This PR is for the 3.5, you don't need to touch older upgrade scripts. At this point your new field will not exist yet.